WordPress/Classes/Categories/NSURLRequest+Helpers.h

```diff
@@ -0,0 +1,14 @@
+#import <Foundation/Foundation.h>
+
+@interface NSURLRequest (Helpers)
```

Comment:
This is pretty minor, but it may make more sense to create a subclass (such as `WPURLRequest`) instead of a category, since the code in `requestForAuthenticationWithURL` seems to be WP-specific / non-generic.